lib/client-manager.js

```diff
@@ -15,7 +15,8 @@ function ClientManager (opts) {
     // only make requests to external sites every 15 seconds.
     ttl: 15,
     // how long should we cache if request to upstream server failed?
-    ttlFailure: 2
+    ttlFailure: 2,
+    packageDoc: require('./package-doc')()
```

Comment:
Would you consider passing `opts` to `require('./package-doc')(opts)` a bad practice? It would conflate which options apply to which layer/instance, but it would also make injection for testing easier. Asking more for my own benefit here based on other code I've written.